test/chained_direct_deps/BUILD

```diff
@@ -0,0 +1,42 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
if these don't need to be public, can remove this

Reply:
Done. I set it public because that was also the case for the other test build file but there is really no reason for it.